clusterloader2/testing/load/config.yaml

```diff
@@ -15,28 +18,63 @@
 {{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
 {{$mediumRCsPerNamespace := DivideInt $mediumRCs $namespaces}}
 {{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
-{{$totalPodsRounded := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
-# saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
-{{$saturationQPS := DivideFloat (AddInt $bigRCs $mediumRCs $smallRCs) (DivideFloat $totalPodsRounded 10)}}
+{{$totalPods := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
+# saturationTime - time for RC saturation should be an equivalent to 10 [pods/s] qps.
+{{$saturationTime := DivideFloat $totalPods $PODS_SATURATION_QPS}}
 #Test
 name: load
 automanagedNamespaces: {{$namespaces}}
 tuningSets:
-- name: AvgSaturationQPS
-  randomizedLoad:
-    averageQps: {{$saturationQPS}}
-- name: AvgScalingQPS
-  randomizedLoad:
+- name: SvcSaturationQPS
+  qpsLoad:
+    qps: {{$SVC_SATURATION_QPS}}
+- name: RandomizedSaturationTimeLimited
+  RandomizedTimeLimitedLoad:
+    timeLimit: {{MultiplyInt $saturationTime $SECOND_TO_NANO}}
```

Comment:
Please don't.
 API should be reasonable. Passing timeouts in nanoseconds is not reasonable.

Seconds is what we should pass.

Reply:
I know, it ugly. But that's how goalng  parses time.Duration using json deserializer.
I haven't found any clean solution for this problem yet:
- Creating additional type that implements Marshal and Unmarshal interfaces. Requires to be cast to time.Duration when used.
- Implementing new json encoder. Sounds like a bad idea.
- Having one api for serializing and other for internal use. Also bad idea.

Reply:
Maybe we should use string representation of duration?
I think that then you san do something like "5s" for example.

Reply:
I guess what i would like to have is something like this:
https://github.com/kubernetes/perf-tests/pull/242/files#diff-8b375d89a1280308d37301996c699680R45

Reply:
Done. Changes from #277 applied.